runtime/src/qio/qbuffer.c

```diff
@@ -148,14 +148,8 @@ qioerr _qbytes_init_iobuf(qbytes_t* ret)
 {
   void* data = NULL;
   
-  // allocate 4K-aligned (or page size aligned)
-  // multiple of 4K
-  data = qio_valloc(qbytes_iobuf_size);
+  data = qio_memalign(sys_page_size(), qbytes_iobuf_size);
```

Comment:
Up to you. I think we should convert it to sys_memalign but that is more of a behavior change than you probably want to do in this PR.